client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -72,6 +73,20 @@ export const requestJetpackOnboardingSettings = ( { dispatch, getState }, action
 	);
 };
 
+export const announceRequestFailure = ( { dispatch, getState }, { siteId } ) => {
+	const url = getUnconnectedSiteUrl( getState(), siteId );
+	const noticeOptions = {
+		id: `jpo-communication-error-${ siteId }`,
+	};
+
+	if ( url ) {
+		noticeOptions.button = translate( 'Visit site admin' );
+		noticeOptions.href = trailingslashit( url ) + 'wp-admin/admin.php?page=jetpack';
+	}
+
+	return dispatch( errorNotice( translate( 'Something went wrong.' ), noticeOptions ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Oops, something went wrong.' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jpo-error-notice-when-communication-error)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).